tokens/psp22/5HZxA385SYeydqZUpTeKj7D37T1bL9N6JA7Xde5QMP8qiSym/info.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "Cats are Liquid",
+  "symbol": "NUKO",
+  "type": "PSP22",
+  "contractAddress": "5HZxA385SYeydqZUpTeKj7D37T1bL9N6JA7Xde5QMP8qiSym",
+  "decimals": 12,
+  "description": "$NUKO is the first cat meme token on @Aleph__Zero",
+  "website": "https://www.nukotoken.org/",
+  "sourceCode": "https://github.com/takeshi-nuko/PSP22-NUKO/",
+  "links": [
+    {
+      "name": "X/Twitter",
+      "url": "https://x.com/nukotoken"
+    },
+    {
+      "name": "Telegram",
+      "url": "https://t.me/nukotoken"
+    },
+    {
+      "name": "Discord",
+      "url": ""
+    }
+  ],
+  "customTags": ["$NUKO Contract Address"],
+  "warnings": [],
+  "owner": "5HC2nukon6y82uby553npmLVf5MnN1faXjLPqVKUhhY2EJQt",
```

Comment:
Hi -- after I inspected the code it seems to me the contract does not have any owner or minter functionality. Is the assessment correct?

If yes, you can leave "owner" and "minter" empty, like:
```
"owner": "",
"minter": ""
```
Contracts with active owners/minters will have warnings displayed on ascan, so you definitely want to avoid that.

Reply:
I had mistakenly thought that owner and minter referred to the account that deployed the project.
As you pointed out, I have left these boxes empty.
Thank you for your advice.